content/agents/overview.md

```diff
@@ -68,6 +68,7 @@ However, you should be aware of certain implications:
 
 * SSH port (22 by default) should be open for incoming connections.
 * An SSH server should be running on the host.
+* The agent user must have sudo privileges.
```

Comment:
Not sure what's wrong with the wording, The "agent user" refers to the user running on the VM, not only for remote installation but also for running the agent services.